lib/Dialect/Torch/Transforms/ReduceOpVariants.cpp

```diff
@@ -39,68 +39,81 @@ class ConvertToImmutableTensors : public RewritePattern {
         opOperand.set(rewriter.create<CopyToValueTensorOp>(op->getLoc(),
                                                            opOperand.get()));
       } else if (auto listType = operandType.dyn_cast<ListType>()) {
-        if (!listType.getContainedType().isa<NonValueTensorType>())
```

Comment:
Can you fix the formatting in this file? It's hard to tell which changes are new and which are not